crates/burn-common/src/id.rs

```diff
@@ -1,45 +1,33 @@
-use alloc::string::String;
-
 use crate::rand::gen_random;
 
-use data_encoding::BASE32_DNSSEC;
-
 /// Simple ID generator.
 pub struct IdGenerator {}
 
 impl IdGenerator {
     /// Generates a new ID.
-    pub fn generate() -> String {
-        // Generate 6 random bytes (281,474,976,710,656 combinations)
-        let random_bytes: [u8; 6] = gen_random();
-
-        // Encode the random bytes in base32 DNSSEC
-        // 6 bytes encodes to 10 lower case characters, e.g. "3uu5e6vv7c"
-        BASE32_DNSSEC.encode(&random_bytes)
+    pub fn generate() -> u64 {
+        // Generate a random u64 (18,446,744,073,709,551,615 combinations)
+        let random_bytes: [u8; 8] = gen_random();
+        u64::from_le_bytes(random_bytes)
```

Comment:
This might be a breaking change, making old models not able to load. Or do you take care of that in the deserializer?

Reply:
Ah right yes.... What's the procedure here? Do I need to handle upgrading the data somewhere?

Reply:
I'm actually unsure. Maybe we keep the `[u8; 8]` as the type carried by the `ParamId`, and I think `[u8; 8]` actually implements `Copy` so that would be great. Then we can use the right decoding/encoding algorithm during serialization/deserialization.

Reply:
Well it was 6 bytes before! But anyway couldn't find much where changing the type helped. In the end found that there's already a ParamSerde defined that really defines the payload. I've added a custom deserializer in there that can load the old format!

Reply:
Hm deserializing two potentially different formats is actually bit harder - switched to just always serializing / deserializing base32 strings (either 6 or 8 bytes in length).